src/legacy/ui/ui_render/ui_render_mixin.js

```diff
@@ -117,14 +117,18 @@ export function uiRenderMixin(osdServer, server, config) {
               themeVersion === 'v7'
                 ? `${regularBundlePath}/osd-ui-shared-deps/${UiSharedDeps.darkCssDistFilename}`
                 : `${regularBundlePath}/osd-ui-shared-deps/${UiSharedDeps.darkV8CssDistFilename}`,
-              `${basePath}/node_modules/@osd/ui-framework/dist/kui_dark.css`,
+              themeVersion === 'v7'
+                ? `${basePath}/node_modules/@osd/ui-framework/dist/kui_dark.css`
+                : `${basePath}/node_modules/@osd/ui-framework/dist/kui_next_dark.css`,
```

Comment:
Seems undesirable to actually reference `node_modules`. Could we use [`require.resolve`](https://nodejs.org/api/modules.html#requireresolverequest-options) here?

Reply:
If I am not mistaken, these are URLs being formed here and are not being included.

Reply:
I agree that it's a bit weird to explicitly reference these paths. But I don't think it's worth refactoring now, as we'd prefer to just remove the inclusion of these extra stylesheets altogether.